app/models/trainee.rb

```diff
@@ -30,6 +30,8 @@ class Trainee < ApplicationRecord
 
   delegate :update_training_route!, to: :route_data_manager
 
+  delegate :bursary_amount, :can_apply_for_bursary?, :funding_available?, to: :funding_manager
```

Comment:
Its better not to have these methods on Trainee anymore and use the `FundingManager` instead, eg: `trainee.funding_manager.bursary_amount`

Reply:
I would even say not to have `trainee.funding_manager` and when we need it use the single responsibility object like this:

```
funding_manager = FundingManager.new(trainee)
funding_manager.bursary_amount

```

Reply:
done in https://github.com/DFE-Digital/register-trainee-teachers/pull/1429/commits/21d3ed87a3aa08e140e7f7599cd86feab2599c3a

:expressionless: